index.js

```diff
@@ -18,6 +18,28 @@ class BirthdayReminder extends q.DesktopApp {
         const monthOfTheBirthday = this.config.monthOfTheBirthday; // this is the birthday month entered by the user
         const dayOfTheBirthday = this.config.dayOfTheBirthday; // this is the birthday day entered by the user
 
+        const currentTimestamp = new Date(now.getFullYear(), currentMonth, currentDay); // get the current date year/month/day
+        let yearOfTheBirthday = now.getFullYear(); // get the current year if the current date is before the birthday date
+        // get the year if the current date is after or equal to the birthday date:
+        if(monthOfTheBirthday < currentMonth || (monthOfTheBirthday == currentMonth && dayOfTheBirthday < currentDay)) {
+            yearOfTheBirthday += 1;
+        }
+        const birthdayTimestamp = new Date(yearOfTheBirthday, monthOfTheBirthday, dayOfTheBirthday); // exact date of the birthday year/month/day
+        const daysToBirthday = (birthdayTimestamp - currentTimestamp)/(1000*3600*24); // number of days before the birthday
+        const month = new Array();
+        month[0] = "January";
+        month[1] = "February";
+        month[2] = "March";
+        month[3] = "April";
+        month[4] = "May";
+        month[5] = "June";
+        month[6] = "July";
+        month[7] = "August";
+        month[8] = "September";
+        month[9] = "October";
+        month[10] = "November";
+        month[11] = "December";
+
```

Comment:
Use an external function that you can declare outside your class.
function getMonthString(monthNumber) {

}
Also I advice you to do const monthsString = ['January', 'February', ...]